src/Components/Search/SearchBar.tsx

```diff
@@ -262,7 +304,7 @@ export class SearchBar extends Component<Props, State> {
             return this.renderSuggestionsContainer(props, { xs })
           }}
           inputProps={inputProps}
-          onSuggestionSelected={this.onSuggestionSelected}
+          onSuggestionSelected={this.onSuggestionSelected.bind(this)}
```

Comment:
For my understanding: we need to bind to `this` instance so that the `@track` decorators have access to the state/props. Right?

Reply:
Hmm, it must be more than that, because `onFocus` doesn't appear to use state/props, but still needs the `.bind(this)`.

Reply:
@pepopowitz We need `.bind(this)` when you pass a method by reference (as if it were a mere function) and it internally references `this`. I’m not entirely sure how it affects the bound decorator, but the method itself doesn’t appear to refer to `this`.

Reply:
@mzikherman If you’re not using the first argument sent to the `onSuggestionSelected` prop, than perhaps you can do away with it here, rather than having to ignore it twice in the `onSuggestionSelected` method and its decorator.

```js
onSuggestionSelected={(_, selection) => this.onSuggestionSelected(selection)}
```

Reply:
oh true, i can just call it directly here!